.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+group: edge
+sudo: required
+services:
+  - mongodb
+  - docker
+node_js:
+  - "0.10"
+  - "4.4.3"
+before_install: npm install
```

Comment:
There was an issue with this template.  The fh-build template command uses fh-npm to do the install and so it was using fh-npm cache which couldn't find `jquery`.  Running a normal npm install before fh-build fixed this problem.  See this build for the error => https://travis-ci.org/feedhenry-templates/welcome-app/jobs/225921377